packages/core/src/lib/TwaManifest.ts

```diff
@@ -326,6 +337,8 @@ export class TwaManifest {
       features: {},
       shareTarget: TwaManifest.verifyShareTarget(webManifestUrl, webManifest.share_target),
       orientation: asOrientation(webManifest.orientation) || DEFAULT_ORIENTATION,
+      scope: webManifest['scope'],
+      fullScopeUrl: new URL(webManifest['scope'] || '.', webManifestUrl).toString(),
```

Comment:
@andreban If `webManifest['scope']` value is invalid (e.g. `//`) the following error will be thrown: `Uncaught TypeError: Failed to construct 'URL': Invalid URL`. Should we add `scope` value validation? If so, how to check if the `scope` value is valid or not?

Reply:
It seems what we'd want is to catch the error and throw a friendlier message?